src/applications/personalization/dashboard/tests/helpers.unit.spec.js

```diff
@@ -45,7 +45,13 @@ const schemaToConfigIds = {
   definitions: 'N/A',
 };
 
-const excludedForms = new Set(['28-1900', '28-8832', '24-0296', '21-4142']);
+const excludedForms = new Set([
+  '28-1900',
+  '28-8832',
+  '24-0296',
+  '21-4142',
+  '21-526EZ-ALLCLAIMS',
```

Comment:
We could add the missing information that is causing the tests to fail. Ideally none of this would need to be specified in the future, and would be captured in manifest instead.